### PR TITLE
#162633451 Set up an empty React project.

### PR DESCRIPTION
**Pull request by araaliFarooq**
## What does this PR do?
Implements a chore to set up an empty React project.

### Description of Tasks completed
* Set up a React development environment.
* Set up Webpack.
* Set up Babel.

### How to install App dependencies
* ``` npm install ``` to install dependencies.

### How to manually test the App
* ``` npm start ``` to start the server.
* ``` npm build ``` to build the app.

### Relevant PT board stories.
- [#162633451](https://www.pivotaltracker.com/story/show/162633451)

### CODE REVIEW

**Comment by mycok** [comment]
LGTM

**Comment by mubstimor** [comment]
LGTM

**Comment by NabunyaLilian** [comment]
Nice work @araaliFarooq , LGTM

**Comment by fatukunda** [comment]
LGTM

**Comment by jeanjoe** [comment]
LGTM

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "presets": ["@babel/preset-env", "@babel/preset-react"]
+  }
+
```
Comment (mubstimor) [inline .babelrc]: Please format this file properly to get rid of the missing new line warning at the end of the file.

**Comment by mubstimor** [comment]
- PR title is too long, can you make it shorter please.
- Add the npm install command for users to easily understand the set up process.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "react-manual",
```
Comment (fatukunda) [inline package.json]: Please rename this to reflect the project name.

**Comment by Qadriyah** [comment]
LGTM

**Pull request merged**